tests/dtm/test_multi_m0d_process_restart.py

```diff
@@ -303,3 +303,132 @@ def test_delete_after_multi_m0d_restart(self):
 
         self.test_completed = True
         self.log.info("ENDED: Verify delete after multiple m0d restart using pkill")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-47160")
+    def test_delete_after_same_m0d_restart_multiple_times(self):
+        """Verify delete after same m0d restart multiple times using pkill"""
```

Comment:
So we are verifying delete objects right? So:
1. I can see we are doing read/validate and then delete so what happens if read fails? 
2. If we want to run read validate and deletes, lets mention the same

Reply:
test case is just specific to delete. In log statement of step 4 of test, read, validate and delete calls are mentioned. 
just for delete operation, s3bench error parser does not work.